### ISSUE TITLE
BG not working

### ISSUE THREAD

**Issue by rpodric** [nextgens/anti-paywall#4]
Thanks for the extension. BG is a tricky one though, I think, and it's restricted to just one or two articles. This extension doesn't seem to affect it at all.

**Comment by nextgens** [comment]
Right; I will get rid of it in the next release; let me know if you have a bypass that works

**Comment by rpodric** [comment]
Thanks. Unfortunately, I'm not aware of anything for this.
